plugins/targetlocker/dblocker/dblocker.go

```diff
@@ -17,8 +17,8 @@ import (
 	"github.com/facebookincubator/contest/pkg/xcontext"
 
 	// this blank import registers the mysql driver
-	_ "github.com/go-sql-driver/mysql"
 	"github.com/benbjohnson/clock"
+	_ "github.com/go-sql-driver/mysql"
```

Comment:
this is no longer next to its comment (about the blank import).
I guess either delete the comment or move it :)

Reply:
Good catch :)